[2024-03-11T14:00:50.351+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-29T00:00:00+00:00 [queued]>
[2024-03-11T14:00:50.578+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-29T00:00:00+00:00 [queued]>
[2024-03-11T14:00:50.583+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-11T14:00:50.771+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): second_task> on 2021-08-29 00:00:00+00:00
[2024-03-11T14:00:50.816+0000] {standard_task_runner.py:60} INFO - Started process 3052 to run task
[2024-03-11T14:00:50.870+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'second_task', 'scheduled__2021-08-29T00:00:00+00:00', '--job-id', '783', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpn65pdhrr']
[2024-03-11T14:00:50.907+0000] {standard_task_runner.py:88} INFO - Job 783: Subtask second_task
[2024-03-11T14:00:51.371+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.second_task scheduled__2021-08-29T00:00:00+00:00 [running]> on host 26e63b1d0d39
[2024-03-11T14:00:52.719+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-29T00:00:00+00:00'
[2024-03-11T14:00:52.734+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-11T14:00:52.738+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and I ran after task1']
[2024-03-11T14:00:52.865+0000] {subprocess.py:86} INFO - Output:
[2024-03-11T14:00:52.932+0000] {subprocess.py:93} INFO - hey, I am task2 and I ran after task1
[2024-03-11T14:00:52.951+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-11T14:00:53.095+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=second_task, execution_date=20210829T000000, start_date=20240311T140050, end_date=20240311T140053
[2024-03-11T14:00:53.481+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T14:00:53.696+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
